packages/random/random.js

```diff
@@ -7,20 +7,25 @@
 // window.crypto.getRandomValues() or alea, the primitive is fraction and we use
 // that to construct hex string.
 
-if (Meteor.isServer)
-  var nodeCrypto = Npm.require('crypto');
+import { Meteor } from 'meteor/meteor';
+
+let nodeCrypto;
+if (Meteor.isServer) {
+  import crypto from 'crypto';
```

Comment:
Unfortunately, I think this will cause the `crypto` stub (which is very large) to be included in the client JS bundle, since `import` declarations register bundle dependencies but `Npm.require` is ignored by the dependency scanner (it just has to work on the server at runtime).

One possible solution:
```js
if (Meteor.isServer) {
  const cryptoId = "crypto";
  nodeCrypto = require(cryptoId);
}
```

Reply:
Ok, I've done it like that for now, but it seems kinda hacky. What about having a separate main module for client and server that inherit from an abstract RandomGenerator class? Could do away with some conditionals in the methods too.